[terra-signature] Implementation page doc updates (#4033)
cerner/terra-core: packages/terra-core-docs/src/terra-dev-site/doc/signature/About.1.doc.mdx

import Signature from 'terra-signature';
```

## Consuming teams are responsible for

> - **Providing Descriptive Labels**: Ensure that the signature component is labeled clearly and descriptively for screen reader users. This includes providing labels for form fields and instructions for signing.

Reviewer: ```suggestion
- **Providing Descriptive Labels**: Ensure that the terra-signature component is labeled clearly and descriptively for screen reader users. This includes providing labels for form fields and instructions for signing.
```

Author: Updated

> - **Implementing Keyboard Accessibility**: Ensure that all functionality of the signature component can be accessed and operated using a keyboard alone, without relying on mouse interactions.

Reviewer: ```suggestion
- **Implementing Keyboard Accessibility**: Ensure that all functionality of terra-signature can be accessed and operated using only a keyboard, without any mouse interactions.
```

Author: Updated

> - **Using ARIA Attributes**: Use ARIA (Accessible Rich Internet Applications) attributes to enhance the accessibility of the signature component, such as aria-label and aria-describedby, to provide additional context for screen reader users.

Reviewer: ```suggestion
- **Using ARIA Attributes**: Use ARIA (Accessible Rich Internet Applications) [attributes](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA) to enhance the accessibility of terra-signature, such as [aria-label](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label) and [aria-describedby](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-describedby), to provide additional context for screen reader users.
```

Author: Updated

> - **Testing with Assistive Technologies**: Test the signature component with a variety of assistive technologies, such as screen readers, to ensure that it is fully accessible and usable by all users.

Reviewer: ```suggestion
- **Testing with Assistive Technologies**: Test terra-signature with a variety of assistive technologies, such as screen readers, to ensure that it is fully accessible and usable by all users.
```

Author: Updated

> - **Providing Alternative Text for Images**: If the signature component includes any images or icons, provide alternative text that describes the content or function of the image for users who cannot see it.

Reviewer: ```suggestion
- **Providing Alternative Text for Images**: If terra-signature includes any images or icons, provide alternative text that describes the content or function of the image for users who cannot see it.
```

Author: Updated

> - **Ensuring Sufficient Color Contrast**: Ensure that there is sufficient color contrast between text and background colors to make the signature component readable for users with low vision or color blindness.

Reviewer: ```suggestion
- **Ensuring Sufficient Color Contrast**: Ensure that there is sufficient color contrast between text and background colors to make terra-signature readable for users with low vision or color blindness.
```

Author: Updated

> - **Documenting Accessibility Features**: Document the accessibility features of the signature component for end users, including instructions on how to use it with assistive technologies.

Reviewer: ```suggestion
- **Documenting Accessibility Features**: Document the accessibility features of terra-signature for end users, including instructions on how to use it with assistive technologies.
```

Author: Updated
